dashboards/google-vertex-ai/vertex-ai-model-garden.json

```
@@ -1,1023 +1,1148 @@
                     "aggregation": {
                       "alignmentPeriod": "60s",
-                      "perSeriesAligner": "ALIGN_RATE",
                       "crossSeriesReducer": "REDUCE_SUM",
                       "groupByFields": [
                         "resource.label.\"model_user_id\""
-                      ]
-                    }
+                      ],
+                      "perSeriesAligner": "ALIGN_RATE"
+                    },
+                    "filter": "metric.type=\"aiplatform.googleapis.com/publisher/online_serving/consumed_throughput\" resource.type=\"aiplatform.googleapis.com/PublisherModel\""
                   },
-                  "unitOverride": "",
-                  "outputFullDuration": false
-                },
-                "plotType": "STACKED_BAR",
-                "legendTemplate": "",
-                "minAlignmentPeriod": "60s",
-                "targetAxis": "Y1"
+                  "unitOverride": ""
+                }
               }
             ],
             "thresholds": [],
             "yAxis": {
               "label": "",
               "scale": "LINEAR"
-            },
-            "chartOptions": {
-              "mode": "COLOR",
-              "showLegend": false,
-              "displayHorizontal": false
             }
-          },
-          "title": "Token throughput by model"
+          }
         }
       },
       {
-        "yPos": 126,
-        "width": 48,
+        "yPos": 178,
         "height": 4,
+        "width": 48,
         "widget": {
-          "title": "Tokens and characters",
+          "title": "Provisioned Throughput",
           "sectionHeader": {
-            "subtitle": "Use the token and character throughput signals to estimate costs and to compare against your quota usage.",
-            "dividerBelow": false
+            "dividerBelow": false,
+            "subtitle": "Provisioned Throughput is a fixed-cost monthly subscription or weekly service that reserves throughput for supported generative AI models on Vertex AI."
           }
         }
       },
       {
-        "xPos": 24,
-        "yPos": 130,
-        "width": 24,
-        "height": 16,
+        "yPos": 182,
+        "height": 17,
+        "width": 29,
         "widget": {
-          "title": "",
-          "singleViewGroup": {
-            "displayType": "DROPDOWN"
-          }
-        }
-      },
-      {
-        "xPos": 24,
-        "yPos": 130,
-        "width": 24,
-        "height": 16,
-        "widget": {
-          "title": "Token throughput by input vs output",
+          "title": "Provisioned Throughput Utilization",
           "xyChart": {
+            "chartOptions": {
+              "displayHorizontal": false,
+              "mode": "COLOR"
+            },
             "dataSets": [
               {
+                "legendTemplate": "${resource.labels.model_user_id} ${resource.labels.location}${resource.labels.monitored_resource_container}",
```

It may also improve readability if one of the 3 components is in brackets. The resource.labels.location can perhaps be removed if the monitored_resource_container may already include the value.

```suggestion
                "legendTemplate": "${resource.labels.model_user_id} ${resource.labels.location} ${resource.labels.monitored_resource_container}",
```